utils/playbackProfiles/helpers/fmp4AudioFormats.ts

```diff
@@ -35,7 +36,9 @@ export function getSupportedFmp4AudioCodecs(
   }
 
   if (getSupportedAudioCodecs('flac')) {
-    codecs.push('flac');
+    if (!browserDetector.isEdge()) {
+      codecs.push('flac');
+    }
```

Comment:
Why? Version 16 should support it: https://caniuse.com/flac. Maybe we should call ``canPlay`` instead of hardcoding this?

Reply:
It's indeed supported in MP4 and MKV container.
But I tested in Edge 88 and FLAC is not supported in shaka-player with FMP4-HLS.